core/common/src/main/java/alluxio/resource/LockResource.java

```diff
@@ -44,4 +44,12 @@ public LockResource(Lock lock) {
   public void close() {
     mLock.unlock();
   }
+
+  /**
+   * ReadWriteLock types.
+   */
+  public enum LockMode {
```

Comment:
remove?

Reply:
I moved `LockMode` from `alluxio.master.file.meta.InodeTree.LockMode` to here, because this way, we won't need dependency on the master module to use LockMode. I feel LockMode belongs in the common module more than master module.